test/groups.yaml

```diff
@@ -94,6 +94,27 @@ localvolumeprovisioner:
     - "defaultstorageclass-protection"
     - "localvolumeprovisioner"
 
+# ------------------------------------------------------------------------------
+# AWS
+#
+# All AWS utility addons should be tested as a part of this group
+# ------------------------------------------------------------------------------
+aws:
+    - "defaultstorageclass-protection"
+    - "cert-manager"
```

Comment:
Is `cert-manager` required for `awsebscsiprovisioner` ?

Reply:
Yes